### PR TITLE
Use the given table name on migrations

### PR DESCRIPTION
**Pull request by m1guelpf**

### CODE REVIEW

**Comment by clarkeash** [comment]
Hi, Thanks for your contribution.

The Invite model also needs to use this value, if you update the model I will look into merging this PR

**Comment by m1guelpf** [comment]
@clarkeash Done!

**Comment by clarkeash** [comment]
Tests have failed. You cant use function calls to set default values for properties.

You will have to set the table name in in the models constructor 😄

**Comment by clarkeash** [comment]
@m1guelpf what happened?

**Comment by m1guelpf** [comment]
@clarkeash I searched a bit and didn't find another way to set the table name of a model

**Comment by clarkeash** [comment]
You can just do it in the constructor:

```php
public function __construct()
{
    $this->table = config('doorman.invite_table_name');
}
```

**Comment by m1guelpf** [comment]
@clarkeash That wouldn't work as the `$table` value is overwritten later. I've reviewed the core and I think the setTable function may work.

**Comment by clarkeash** [comment]
overwritten by what?

**Comment by clarkeash** [comment]
and setTable is fine, but you still need to do this in the constructor, you cant just call a method like that in PHP

**Comment by m1guelpf** [comment]
@clarkeash I'll fix that in a moment

File: src/Models/Invite.php [hunk 1]
```diff
@@ -7,4 +7,10 @@
 class Invite extends Model
 {
     protected $dates = [ 'valid_until' ];
+    
+    public function __construct(array $attributes = [])
```
Comment (m1guelpf) [inline src/Models/Invite.php:11]: @clarkeash Do you think the `$attributes` array should be removed?
I copied it from [Eloquent's __construct()](https://github.com/laravel/framework/blob/master/src/Illuminate/Database/Eloquent/Model.php#L139)

Comment (clarkeash) [reply src/Models/Invite.php:11]: keeping it is correct

**Pull request merged**